### PR TITLE
Upgrade to netcoreapp22

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by ajcvickers** [comment]
@ryanbrandenburg Given that EF 2.1+ targeting netcoreapp2.0 is still a valid scenario, is this something that has to be done on the EF repo?

File: benchmarks/EFCore.Benchmarks.EFCore/EFCore.Benchmarks.EFCore.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
```
Comment (smitpatel) [inline benchmarks/EFCore.Benchmarks.EFCore/EFCore.Benchmarks.EFCore.csproj]: No modifications should be done to any project under benchmarks folder.

Comment (ryanbrandenburg) [reply benchmarks/EFCore.Benchmarks.EFCore/EFCore.Benchmarks.EFCore.csproj]: Can you go into a bit more detail than that? Why not? Why do we test these benchmarks against netcoreapp2.0 but not netcoreapp2.1 currently?

**Comment by ryanbrandenburg** [comment]
The decision was made in engineering triage to only have net461 and netcoreapp2.2 as TFM's in tests and samples. If you have specific reasons why this shouldn't be the case for EF lets start an email thread.

**Comment by ajcvickers** [comment]
@ryanbrandenburg What I'm really asking is what is broken if we don't remove netcoreapp2.0 on EF?

**Pull request closed without merging**